### PR TITLE
genesys: usbhub: Fix test for static_tool_string value

### PR DESCRIPTION
**Pull request by gportay**
Additionally, this regroups the call of a function and the test for the
returned value.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x] Code fix
- [ ] Feature
- [ ] Documentation

This fixes a minor issue that has escaped from my review :facepalm:.

### CODE REVIEW

**Comment by gportay** [comment]
@adamgene could you confirm this is a fix?

**Review by hughsie — APPROVED** [review]

File: plugins/genesys/fu-genesys-usbhub-device.c [hunk 1]
```diff
@@ -954,7 +954,7 @@ fu_genesys_usbhub_device_setup(FuDevice *device, GError **error)
 				    self->dynamic_ts.bonding);
 			return FALSE;
 		}
-		if (self->static_ts.tool_string_version < TOOL_STRING_VERSION_BONDING_QC)
```
Comment (gportay) [inline plugins/genesys/fu-genesys-usbhub-device.c:957]: That condition is never met as `self->static_ts.tool_string_version` is at least `0x30` (aka `'0'`) and `TOOL_STRING_VERSION_BONDING_QC` is `2`.

**Comment by gportay** [comment]
@hughsie this can be merged, I have tested it and got the value I expect with this change.

File: plugins/genesys/fu-genesys-usbhub-device.c (hunk 1 above)
Comment (adamgene) [reply plugins/genesys/fu-genesys-usbhub-device.c:957]: Yes, you are right.

Comment (gportay) [reply plugins/genesys/fu-genesys-usbhub-device.c:957]: cool, thanks.

**Pull request merged**